docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  wdio-cucumber:
+    build: .
+    image: wdio-cucumber
+    container_name: wdio-cucumber
+    links:
+      - selenium
+
+  selenium:
+    image: selenium/standalone-chrome
```

Comment:
This will just pull the latest version.  Probably should be updated to select which version of the container you'd prefer so we can ensure that the latest version doesn't break anything in future.